common/src/main/resources/META-INF/openapi.json

```diff
@@ -1,2286 +1,2290 @@
-            },
-            "Conflict": {
-                "content": {
-                    "application/json": {
-                        "schema": {
-                            "$ref": "#/components/schemas/Error"
-                        },
-                        "examples": {
-                            "ConflictExample": {
-                                "value": {
-                                    "error_code": 409,
-                                    "message": "The artifact content was invalid."
-                                }
-                            }
-                        }
-                    }
-                },
-                "description": "Common response used when an input conflicts with existing data."
-            }
-        }
-    },
-    "tags": [
-        {
-            "name": "Artifacts",
-            "description": "The primary way to interact with the Apicurio Registry API is to add, update, \nor delete artifacts. This section includes all of these primary operations."
-        },
-        {
-            "name": "Metadata",
-            "description": "Sometimes the metadata for an artifact is important. For example, metadata includes \nwhen the artifact was created, last updated, and so on. This section contains \noperations to access (and in some cases change) an artifact's metadata."
-        },
-        {
-            "name": "Versions",
-            "description": "Whenever an artifact's content is updated, the old versions of the artifact's content\nare not lost.  All versions can be listed and accessed if necessary.  This section \ndescribes the operations used to list and access all versions of an artifact's content\nand metadata."
-        },
-        {
-            "name": "Artifact rules",
-            "description": "Rules can be configured on a per-artifact basis, allowing for different approaches\nto content evolution for each artifact.  These rules will override any global rules\nthat have been configured.  This section contains the operations used to manage a\nsingle artifact's rules."
-        },
-        {
-            "name": "Global rules",
-            "description": "Global rules can be configured in the registry to govern how artifact content can \nevolve over time (as artifact content is **updated**). Global rules will be applied \nwhenever an artifact is added to the registry, and also whenever an artifact's \ncontent is updated (only if that artifact doesn't have its own specific rules \nconfigured). This section describes the operations used to manage the global rules."
-        },
-        {
-            "name": "Search",
-            "description": "The search API is used to browse or find artifacts in the registry. This section describes the operations for searching for artifacts and versions. "
-        }
-    ]
-}
+{
+    "openapi": "3.0.2",
+    "info": {
+        "title": "Apicurio Registry API",
+        "version": "1.2.2",
```

Comment:
@EricWittmann it would be nice to update version to the current one (1.3.1-SNAPSHOT) also and keep it in sync on release with the new snapshot version

Reply:
I've already submitted a PR to make that change in the release workflow.  I am not setting it to the -SNAPSHOT version after the release is done, though.  But at least it will be right each time a release is performed.